lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskQueryMapper.java

```diff
@@ -37,6 +40,9 @@
           + "<if test=\"joinWithAttachmentClassifications\">"
           + "LEFT JOIN CLASSIFICATION AS ac ON a.CLASSIFICATION_ID = ac.ID "
           + "</if>"
+          + "<if test=\"addWorkbasketNameForOrdering\">"
```

Comment:
I'm missing a separate "joinWithWorkbaskets" property here like for the attachments and classifications.

Reply:
explained above